popup-basic/definition.json

```diff
@@ -337,9 +337,16 @@
         },
         {
           "name": "deeplink_url",
+          "default": "myapp://path/to/content or http://your.url",
           "label": {
-            "en": "Deeplink Url",
-            "fr": "Url de Deeplink"
+            "en": "URL",
+            "fr": "URL"
+          },
+          "type": "string",
+          "tooltip": {
+            "en": "App link or web page",
+            "fr": "Lien dans l'app ou page web"
+          }
```

Comment:
there's a mistake here with the brackets. Please check that you have linter errors showing when building JSONs.